test/helpers.js

```diff
@@ -386,14 +386,66 @@ export const expectResponseToContainNecessaryVendorData = (data) => {
   expect(data).to.have.property('address');
 };
 
-export const expectResponseToContainNecessaryPropertyData = (data) => {
-  expect(data).to.have.property('name');
-  expect(data).to.have.property('address');
-  expect(data).to.have.property('mainImage');
-  expect(data).to.have.property('gallery');
-  expect(data).to.have.property('price');
-  expect(data).to.have.property('houseType');
-  expect(data).to.have.property('description');
+export const expectResponseToContainNecessaryPropertyData = (data, object) => {
```

Comment:
With the current naming, it is difficult to differentiate the response data from the actual property data.
```suggestion
export const expectResponseToContainNecessaryPropertyData = (response, property) => {
```